EXILED/Exiled.API/Features/Roles/FpcRole.cs

```diff
@@ -55,6 +58,19 @@ public RelativePosition RelativePosition
             set => FirstPersonController.FpcModule.Motor.ReceivedPosition = value;
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether if the player should get <see cref="Enums.DamageType.Falldown"/> damage.
+        /// </summary>
+        public bool IsFallDamageEnable
+        {
+            get => FirstPersonController.FpcModule.Motor._enableFallDamage;
+            set
+            {
+                enableFallDamageField ??= AccessTools.Field(typeof(FpcMotor), nameof(FpcMotor._enableFallDamage));
```

Comment:
No publicized?

Reply:
It's get only

Reply:
was readonly `private readonly bool _enableFallDamage;` it's not publicized